lib/class/api.class.php

```diff
@@ -1966,7 +1966,7 @@ public static function user_update($input)
         $password   = $input['password'];
         $state      = $input['state'];
         $city       = $input['city'];
-        $disable    = ($input['disable'] == 'true' ? true : false;
+        $disable    = $input['disable'] == 'true' ? true : false;
```

Comment:
It's always best practice not to mix coding styles. You always used braces `( ... )` around inline comparisons. Why not here?
In my opinion it also makes it easier to read.

Reply:
They do. and that's changed. added a few missing ones as well.